projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -201,7 +201,7 @@
 			var targetPosition = Math.max(
 				0,
 				el.offsetTop -
-					Math.max( 0, window.innerHeight - ( extraInfoContentAreaHeight + 64 + 15 + 35 ) ) // Subtract footer height plus content area padding.
+					Math.max( 0, window.innerHeight - ( extraInfoContentAreaHeight + 64 + 35 + 35 ) ) // Subtract footer height plus content area padding.
```

Comment:
Could we programatically grab the footer height and content area padding here? This will almost certainly get missed if this is changed in the CSS in the future.

Reply:
Oh, yes, thanks for the note. I'd been trying to avoid the function from getting too long, but agreed, it's much better to calculate based on the real positions 👍

Reply:
I've updated this to get the real values from the DOM when we begin the scroll, and have moved the calculations to a separate function to try to tidy things up a little bit.